pyproject.toml

```diff
@@ -28,7 +28,7 @@ scikit-image = "*"
 fuzzywuzzy = "*"
 PyWavelets = "1.3.0"
 networkx = "*"
-pymedphys = ">=0.35,<=0.37.1"
+pymedphys = ">=0.35,<=0.39.3"

```

Comment:
whoops, no.  This allows any of the pymedphys versions between 0.36 and 0.39.3, and it should only allow 0.39.3 and later.  I specified it so that it would use 0.39.3 up to but not including 0.40, if you want to specify it to just be >= 0.39.3